### PR TITLE
List cases with missing HPO terms, pheno groups and blank synopsis

### PR DESCRIPTION
**Pull request by northwestwitch**
In cases page, write 'synopsis:', 'HP:' or 'PG:' to get all cases with missing info for these fields

### CODE REVIEW

**Comment by northwestwitch** [comment]
fix #1198. It's on stage.
In the beginning I was thinking that the code was not working, then I realized that A LOT of those test cases don't have phenotype terms, and that's why I kept getting the same list even if I used the filter!

File: scout/server/blueprints/cases/templates/cases/cases.html [hunk 1]
```diff
@@ -119,7 +119,8 @@ <h5 class="modal-title" id="sangerModalLabel">Sanger validations to evaluate:</h
       </div>
     </div>
     <div class="bottom-align-text">
-      <br>Examples: case_id=<strong>18201</strong>, HPO-term=<strong>HP:0001166</strong>, synopsis=<strong>synopsis:epilepsy</strong>, panel=<strong>panel:NMD</strong>, phenotype group=<strong>PG:0100022</strong>, cohort=<strong>cohort:pedhep</strong>.
+      <br>Examples: case_id=<strong>18201</strong>, HPO-term=<strong>HP:0001166</strong>, synopsis=<strong>synopsis:epilepsy</strong>, panel=<strong>panel:NMD</strong>, phenotype group=<strong>PG:0100022</strong>, cohort=<strong>cohort:pedhep</strong>.<br>
+      Leave blank values after key to search for cases with missing info (<strong>HP:</strong>,<strong>PG:</strong>,<strong>synopsis:</strong>).
```
Comment (dnil) [inline scout/server/blueprints/cases/templates/cases/cases.html]:
```suggestion
      Use key without value to search for cases with missing info (<strong>HP:</strong>,<strong>PG:</strong>,<strong>synopsis:</strong>).
```

**Comment by dnil** [comment]
What do you think: do we need to test this (and the other name search PR) on stage separately? Perhaps we could merge them and then test deployment?

**Comment by northwestwitch** [comment]
Yes, we can test them togethers I'd say, but maybe write a test for #1210 ?

**Pull request merged**